app/models/rent.rb

```diff
@@ -6,13 +6,11 @@ class Rent < ApplicationRecord
   validates :start_period, presence: true, if: :valid_start_date?
   validates :end_period, presence: true, if: :valid_end_date?
 end
-
-# Implement the method example 
+ 
 def valid_start_date?
-  true
+  :start_period >= Date.today
 end
 
-# Implement the method example 
 def valid_end_date?
-  true
+  :end_period >= :start_period
```

Comment:
just a small comment- shouldn't it be strict comparison, i.e. `:end_period > :start_period` ?

Reply:
Fixed, thank you!